pkg/agent/agent.go

```diff
@@ -1141,3 +1104,82 @@ func (i *Initializer) patchNodeAnnotations(nodeName, key string, value interface
 func (i *Initializer) getNodeInterfaceFromIP(nodeIPs *utilip.DualStackIPs) (v4IPNet *net.IPNet, v6IPNet *net.IPNet, iface *net.Interface, err error) {
 	return getIPNetDeviceFromIP(nodeIPs, sets.NewString(i.hostGateway))
 }
+
+func (i *Initializer) initLocalConfig() error {
```

Comment:
I think we should keep it to be initNodeLocalConfig, and add a new func called initK8sNodeLocalConfig()